v2/README.md

```
@@ -41,12 +41,18 @@ $ forge snapshot
 $ anvil
 ```
 
-### Deploy using script
+### Deploy
 
 ```shell
 $ forge script script/<DeployScript>.s.sol:<DeployScript> --rpc-url <your_rpc_url> --private-key <your_private_key>
 ```
 
+##### More detailed instructions
```

Markdownlint

Expected: h4; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)